config/secrets.yml

```diff
@@ -1,7 +1,12 @@
 development:
+  active_record_encryption:
+    primary_key: 5iYPUMb6NWxlsyiRGyBrVgfY6ZPhm5ZA
+    key_derivation_salt: q8sHPMCYthvtuM5N7mv2438my5v4DPaP
   secret_key_base: 101615e3369d108c13f7182caf9bb988fc8f2d8d309ebd16d39b34bece4b4bd0944df576bfa2ff35984e7c447658cc25810540b50759c15c2b94f8ef26867a8a
 test:
   secret_key_base: 3fb6b8dc769442c5f268a1fc6d1238e80bd6cf07b240d02ae622bd635a79f19342363291f053c80ac5f0132fe773f73e561022a46324a1310d9f0d368414d369
 production:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
   notify_api_key: <%= ENV["GOVUK_NOTIFY_API_KEY"] %>
+  primary_key: <%= ENV["ACTIVE_RECORD_PRIMARY_KEY"] %>
```

Comment:
Looks like this is missing an active_record_encryption nesting?

also I think the environment variable is missing the word encryption. It should be ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY